scripts/wmata_data/distcalc.py

```diff
@@ -0,0 +1,149 @@
+import sys
+import csv
+import requests
+
+#TODO replace these with code4dc keys
+WMATA_HEADERS = {'api_key': ''}
+MAPBOX_PARAMS = {'access_token': ''}
+
```

Comment:
Sounds like a good intermediate approach. In the refactored branch, `scripts` gets moved down one level, so it will end up in `repodirectory/python/scripts/wmata_data`. Most of the Python code in the refactored approach actually ends up in a package... probably easier to show you those details tonight.